Wireframe/style.css

```diff
@@ -30,16 +30,31 @@ body {
   background: var(--paper);
   color: var(--ink);
   font: var(--font);
+
 }
+ .title-container{text-align:center ;}
+header{ border: 1px solid grey;
```

Comment:
We generally try to keep consistent indenting in files to make it easier to read them at a glance.

Both of these lines don't follow the indenting of the rest of the file - can you fix up the whole file?